applications/serial_lte_modem/src/slm_at_fota.c

```diff
@@ -53,9 +50,6 @@ enum fota_stage slm_fota_stage = FOTA_STAGE_INIT;
 enum fota_status slm_fota_status;
 int32_t slm_fota_info;
 
-static char path[FILE_URI_MAX];
```

Comment:
Fota tests are not part of the SLM integration tests, so I tried to run them manually, which failed with unknown image type.

Please try to run SLM application for 9151 with following input:
at+cfun=1
at#xfota=1,"http://files.testncs.com/test/slm/9151_477c5b8d_app_update.bin"

This will show unknown DFU target through RTT. With main, the same starts the fota.

Reply:
Thanks, I've just gotten twister tests to pass, so there is likely some quirks to figure out yet.

Reply:
With the latest tests, the fota package is downloaded, however, when the download is completed, SLM fota operation fails (with errno ENOENT (2)) and <err> fota_download: Download client error event.

Reply:
@eivindj-nordic: This actually works with the latest changes. Nice.